cmd/crc/cmd/config/get.go

```diff
@@ -18,15 +19,19 @@ func configGetCmd(config config.Storage) *cobra.Command {
 				return errors.New("Please provide a configuration property to get")
 			}
 			key := args[0]
+
 			v := config.Get(key)
-			switch {
-			case v.Invalid:
+			if v.Invalid {
 				return fmt.Errorf("Configuration property '%s' does not exist", key)
-			case v.IsDefault:
+			}
+
+			telemetry.SetContextProperty(cmd.Context(), "key", args[0])
```

Comment:
We send the key in the "value unset" case, but not in the "non-existing value" case, I would either send in both cases, or in neither.